Loads/Serialization/JsonSerializerSettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Newtonsoft.Json;
+
+namespace MagmaWorks.Taxonomy.Serialization
+{
+    internal static class TaxonomyJsonSerializer
```

Comment:
Should this live somewhere else?

Reply:
Yes it's duplicated a lot - where can it live? A common serialization project? I've kept the serialization per project so the transient dependencies doesnt pull in every single library in taxonomy